includes/admin/tools/class-edd-tools-recount-single-customer-stats.php

```diff
@@ -145,49 +107,16 @@ public function set_properties( $request ) {
 	public function process_step() {
 
 		if ( ! $this->can_export() ) {
-			wp_die( __( 'You do not have permission to modify this data.', 'easy-digital-downloads' ), __( 'Error', 'easy-digital-downloads' ), array( 'response' => 403 ) );
+			wp_die( esc_html__( 'You do not have permission to modify this data.', 'easy-digital-downloads' ), esc_html__( 'Error', 'easy-digital-downloads' ), array( 'response' => 403 ) );
 		}
 
-		$had_data = $this->get_data();
-
-		if( $had_data ) {
-			$this->done = false;
-			return true;
```

Comment:
This method will no longer ever set `$this->done` to `false` like it does here.

It also always appears to always return `false`.

Is that OK?

Reply:
Updated to be more consistent with how it was and the bulk customer recount tool. I am not sure it matters because the progress bar doesn't actually show anything either way, since the new recounting is not incremental.